README.md

```diff
@@ -198,7 +198,7 @@ in your build script.
 ### Pre-processing and post-processing
 
 Setting `onStart` and/or `onEnd` in a build config allows you to hook into the esbuild cycle.
-`onStart(config)` is called when a build starts and `onEnd(config, result)` when a build finishes.
+`onStart(config, changedFiles)` is called when a build starts and `onEnd(config, result)` when a build finishes.
```

Comment:
Please put "when a build finishes" on a new line

Reply:
Done and done!